apps/passport-server/test/generic-issuance/pipelines/auto-issuance/autoIssuanceSpec.ts

```diff
@@ -0,0 +1,1089 @@
+        pipeline.issuanceCapability.options.feedId,
+        ZUPASS_EDDSA_PRIVATE_KEY,
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail,
+        AutoIssuanceBouncerIdentity
+      );
+      expectLength(bouncerTickets, 2);
+      const bouncerTicket = bouncerTickets[0];
+      expectToExist(bouncerTicket);
+      expectIsEdDSATicketPCD(bouncerTicket);
+      expect(bouncerTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+      );
+      // Bouncer ticket is checked out
+      expect(bouncerTicket.claim.ticket.isConsumed).to.eq(false);
+
+      const bouncerPODTicket = bouncerTickets[1];
+      expectToExist(bouncerPODTicket);
+      expectIsPODTicketPCD(bouncerPODTicket);
+      expect(bouncerPODTicket.claim.ticket.attendeeEmail).to.eq(
+        pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+      );
+      // Bouncer ticket is checked out
+      expect(bouncerPODTicket.claim.ticket.isConsumed).to.eq(false);
+    }
+    {
+      // Now check the bouncer in
+      const autoIssuanceCheckinRoute =
+        pipeline.checkinCapability.getCheckinUrl();
+
+      const bouncerCheckInBouncer = await requestCheckInPipelineTicket(
+        autoIssuanceCheckinRoute,
+        ZUPASS_EDDSA_PRIVATE_KEY,
+        bouncerTicket.claim.ticket.attendeeEmail,
+        AutoIssuanceBouncerIdentity,
+        bouncerTicket
+      );
+      expect(bouncerCheckInBouncer.value).to.deep.eq({ success: true });
+      MockDate.set(Date.now() + ONE_SECOND_MS);
+
+      // Reload the pipeline
+      await pipeline.load();
+      {
+        const bouncerTickets = await requestTicketsFromPipeline(
+          pipeline.issuanceCapability.options.feedFolder,
+          autoIssuanceTicketFeedUrl,
+          pipeline.issuanceCapability.options.feedId,
+          ZUPASS_EDDSA_PRIVATE_KEY,
+          pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail,
+          AutoIssuanceBouncerIdentity
+        );
+        expectLength(bouncerTickets, 2);
+        const bouncerTicket = bouncerTickets[0];
+        expectToExist(bouncerTicket);
+        expectIsEdDSATicketPCD(bouncerTicket);
+        expect(bouncerTicket.claim.ticket.attendeeEmail).to.eq(
+          pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+        );
+        // User is now checked in
+        expect(bouncerTicket.claim.ticket.isConsumed).to.eq(true);
+
+        const bouncerPODTicket = bouncerTickets[1];
+        expectToExist(bouncerPODTicket);
+        expectIsPODTicketPCD(bouncerPODTicket);
+        expect(bouncerPODTicket.claim.ticket.attendeeEmail).to.eq(
+          pretixBackend.get().autoIssuanceOrganizer.autoIssuanceBouncerEmail
+        );
+        // User is now checked in
+        expect(bouncerPODTicket.claim.ticket.isConsumed).to.eq(true);
+      }
+    }
+  });
+
+  step(
+    "Pretix should not load tickets for an event with invalid settings",
```

Comment:
Do we need to repeat this test here? It looks identical to the one in the Pretix pipeline tests.

Same applies to some of the tests above and below it.

Reply:
I think I'd rather keep these extra tests in to make sure that this new feature doesn't interact negatively with existing features. I'm not sure about the best way to test the matrix of features that could interact with each other longer term, we probably don't want to be copy pasting stuff like this forever.